internal/gps/vcs_repo.go

```diff
@@ -31,6 +31,25 @@ type gitRepo struct {
 	*vcs.GitRepo
 }
 
+func newGitRepo(ustr, path string) (*gitRepo, error) {
+	r, err := vcs.NewGitRepo(ustr, path)
+	if err != nil {
+		if _, ok := err.(*vcs.LocalError); ok {
+			// if vcs could not initialize the repo due to a local error
+			// then the local repo is in an incorrect state. Remove and
+			// treat it as a new not-yet-cloned repo.
+			os.RemoveAll(path)
```

Comment:
No direct logging from within gps! 📏😄

This is a case where we really want warnings, per #534 (I've already added a bullet item for it). Let's put a TODO in indicating that this should generate a warning, and call it enough for now.